apps/onboarding-ms/src/main/java/it/pagopa/selfcare/onboarding/service/AggregatesServiceDefault.java

```diff
@@ -93,6 +102,16 @@ public Uni<VerifyAggregateResponse> validateAppIoAggregatesCsv(File file) {
                         verifyAggregateAppIoResponse.getErrors().size()));
     }
 
+    @Override
+    public Uni<RestResponse<File>> retrieveAggregatesCsv(String onboardingId, String productId) {
+        return Uni.createFrom().item(() -> azureBlobClient.getFileAsPdf(String.format("%s%s/%s/%s", onboardingMsConfig.getAggregatesPath(), onboardingId, productId, "aggregates.csv")))
```

Comment:
you could use a constant instead hard-coded string, for ex: FILE_NAME_AGGREGATES_CSV